docs/general/reverse-proxy.md

```diff
@@ -0,0 +1,24 @@
+# Reverse Proxy
+
+Within TrueCharts our aim is to make it as easy as possible to secure your Apps. To support this we Supply a Traefik "Reverse Proxy", which has been preconfigured to provide secure and fast connections.
+
+To use Traefik as a Reverse Proxy, all you have to do is enable "Reverse Proxy" in the App of your choice and fill out a little form.
```

Comment:
I'm not totally sure if this is the right target for the link when these docs end up on the website... 

```suggestion
To use Traefik as a Reverse Proxy, all you have to do is enable "Reverse Proxy" in the App of your choice and fill out a little form. See [the Traefik App docs](/apps/traefik/) for more information.
```

Reply:
that would be the right target I think :)